### PR TITLE
Switch to OSP 16.2 by default

### PR DESCRIPTION
**Pull request by EmilienM**
16.2 was released, so let's bump to 16.2 by default instead of 16.1.

### CODE REVIEW

File: playbooks/vars/defaults.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-rhos_release: 16.1
```
Comment (mandre) [inline playbooks/vars/defaults.yaml:2]: Should we perhaps make a copy of the old `defaults.yaml` to an `example-overrides/local-overrides-osp16-1.yaml` file, if someone ever need to deploy RHOSP16.1?

Comment (EmilienM) [reply playbooks/vars/defaults.yaml:2]: done

File: example-overrides/local-overrides-rhel8_2-tripleo-osp16-1.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+standalone_host: <standalone FQDN>
```
Comment (mandre) [inline example-overrides/local-overrides-rhel8_2-tripleo-osp16-1.yaml:1]: You're missing `rhos_release: 16.1` somewhere.

Comment (EmilienM) [reply example-overrides/local-overrides-rhel8_2-tripleo-osp16-1.yaml:1]: this is only needed if you want to install from the puddle. Users who want to use the puddle (and not RHSM) would not need all these bits, just to set `rhos_release: 16.1` without RHSM bits.

**Review by mandre — COMMENTED** [review]
/lgtm

**Pull request merged**